<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseProjectConfigDialog</class>
 <widget class="QDialog" name="CourseProjectConfigDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string> Config for project: Course</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tbCourseProgramme">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/course-project/course-programme.png</normaloff>:/course-project/course-programme.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Programme</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="btnAddProgrammeField">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/plus.png</normaloff>:/course-project/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDeleteProgrammeField">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/minus.png</normaloff>:/course-project/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnRestoreDefaultProgrammeFields">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/list-reset.png</normaloff>:/course-project/list-reset.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="lvProgrammeFields">
         <property name="styleSheet">
          <string notr="true">QListView{show-decoration-selected: 1;} 
QListView::item {padding:5px;}
</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tbCourseTopics">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/course-project/topics.png</normaloff>:/course-project/topics.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Topic Design</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="btnAddTopicDesignField">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/plus.png</normaloff>:/course-project/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDeleteTopicDesignField">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/minus.png</normaloff>:/course-project/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnRestoreTopicDesignFields">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/list-reset.png</normaloff>:/course-project/list-reset.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="lvTopicDesignFields">
         <property name="styleSheet">
          <string notr="true">QListView{show-decoration-selected: 1;} 
QListView::item {padding:5px;}
</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tbCourseLectures">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/course-project/lectures.png</normaloff>:/course-project/lectures.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Lecture Desgin</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="btnAddLectureDesignField">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/plus.png</normaloff>:/course-project/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDeleteLectureDesignField">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/minus.png</normaloff>:/course-project/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnRestoreLectureDesignFields">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/list-reset.png</normaloff>:/course-project/list-reset.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="lvLectureDesignFields">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QListView{show-decoration-selected: 1;} 
QListView::item {padding:5px;}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tbCourseResources">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/course-project/resources.png</normaloff>:/course-project/resources.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Resources Categories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QToolButton" name="btnAddResourceCategory">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/plus.png</normaloff>:/course-project/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDeleteResourceCategory">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/minus.png</normaloff>:/course-project/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnRestoreResourceCategories">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/course-project/list-reset.png</normaloff>:/course-project/list-reset.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="lvResourcesCategories">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QListView{show-decoration-selected: 1;} 
QListView::item {padding:5px;}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CourseProjectConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>CourseProjectConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
